harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -776,11 +944,28 @@ def _load(self, load_path: pathlib.Path) -> None:
                     "callback will be initialized from scratch"
                 )
 
-        # Load workload sequencer state.
         wlsq_path = load_path.joinpath("workload_sequencer.pkl")
```

Comment:
naming issues here.  There should not be an instance of `wlsq` or `workload.*sequencer` in this file after this change, I think.

Reply:
i'm calling the new save state `trial_state` (open to better naming suggestions), but in order to support backwards compatibility, we should still check for wlsq.pkl